numpy/core/tests/test_multiarray.py

```diff
@@ -5001,8 +5001,13 @@ def test_basic(self):
 
             self._clip_type(
                 'uint', 1024, 0, 0, inplace=inplace)
-            self._clip_type(
-                'uint', 1024, -120, 100, inplace=inplace, expected_min=0)
+            if np._get_promotion_state() == "weak":
+                with pytest.raises(OverFlowError, match="Python integer -120"):
+                    self._clip_type(
+                        'uint', 1024, -120, 100, inplace=inplace)
+            else:
+                self._clip_type(
+                    'uint', 1024, -120, 100, inplace=inplace, expected_min=0)
```

Comment:
Ooops, unrelated.  But harmless update to allowing to run the test suite with the new promotion.